wallet/core/src/lib.rs

```diff
@@ -28,15 +33,28 @@
 //! to satisfy the requested amount exceeds the maximum
 //! allowed transaction mass.
 //!
+//! Key generation and derivation is available in the
+//! [`kaspa_wallet_keys`] crate.
+//!
 //! The framework can operate
-//! within native Rust applications as well as within the NodeJS
-//! and browser environments via WASM32.
+//! within native Rust applications as well as within NodeJS, Bun
+//! and browser environments via the WASM32 SDK.
 //!
-//! For JavaScript / TypeScript environments, there are two
+//! WASM32 SDK documentation is available at:
+//! <https://kaspa.aspectron.org/docs/>
+//!
+//! For NodeJS JavaScript and TypeScript environments, there are two
 //! available NPM modules:
 //! - <https://www.npmjs.com/package/kaspa>
 //! - <https://www.npmjs.com/package/kaspa-wasm>
 //!
+//! NOTE: for security reasons (to mitigate potential upstream vendor
```

Comment:
The recommendation here is to **build** from source or **download** from pre-built redistributables as the mitigation. What is the insecure alternative this is trying to warn against?

Reply:
This note is in the NPM module section...  should we rephrase with more focus on NPM?   If you look at this in the actual docs (i.e. clone the repo and run `cd wallet/core && cargo doc --open`), each section of this documentation will appear strictly separated due to H1 headings, so it is much more apparent that this note applies to the NPM block.  I don't think this should be changed.